.github/workflows/pr-e2e-tests.yaml

```diff
@@ -24,6 +24,7 @@ jobs:
         env:
           NEO4J_AUTH: neo4j/password
           NEO4J_ACCEPT_LICENSE_AGREEMENT: yes
+          NEO4J_PLUGINS: fromJson('["apoc"]')
```

Comment:
I think we also need to to update `neo4j.conf` to access the procedures:
`dbms.security.procedures.unrestricted=apoc.*`

Reply:
AFAIK There's no need to do the `fromJson`? The Dockerfile / entrypoint defined in he image take care of everything, including the `dbms.security.procedures.unrestricted=apoc.*`

Reply:
Seems like it doesn't, because the we're getting `ProcedureNotFound` error in the test. Or am I missing something else?

Reply:
What's the procedure being called? Does it work when doing the fromJson?

Reply:
Just checked the neo4j 5 community edition image locally, apoc is neither installed or allowed in the config by default, but providing the `NEO4J_PLUGINS` env var installs the plugin as well as adds the line to the neo4j.conf.

So something is not working with the env var in GitHub 🤔

Reply:
Maybe the version we are using? What are the matrix values?

Reply:
Are we sure we are loading the right image on GitHub? As in, the matrix variables are being used in the service?

Reply:
Yes, according to the logs.

Reply:
I wonder if when doing Docker run Docker takes care of the conversion but GH doesn't do that

Reply:
Or not, I found another syntax that seems to be working! 🎉

Reply:
> I wonder if when doing docker run Docker takes care of the conversion but GH doesn't do that.

Reply:
> Or not, I found another syntax that seems to be working! 🎉 [8855efa](https://github.com/neo4j/neo4j-genai-python/pull/36/commits/8855efa85c067f94d93e43e76aabb428644bb6de)

W-O-W. But basically GH at fault.